spec/unit/outputs/elasticsearch_spec.rb

```diff
@@ -712,6 +711,81 @@
     end
   end
 
+  describe "SSL deprecated settings" do
+    context "with client certificate" do
+      let(:do_register) { true }
+      let(:cacert) { Stud::Temporary.file.path }
+      let(:options) { {
+        "ssl" => "true",
+        "cacert" => cacert,
+        "ssl_certificate_verification" => false
+      } }
```

Comment:
Given that this fixture part is almost equally value in both contexts then it could be lifted in the outer `describe` for example:
```ruby
context "SSL deprecated settings" do
  let(:do_register) { true }
  let(:cacert) { Stud::Temporary.file.path }
  let(:base_options) do  # in case of multiline blocks the do..end should be used instead of {...}
   {
        "ssl" => true,
      }
  end
  
  context "with client certificate" do
    let(:options) do 
      base_options.merge({ 
        "ssl_certificate_verification" => false, 
        "cacert" => cacert,
      }) 
    end   
     # test code here
  end
  
  context "with java stores" do
    let(:options) do base_options.merge({
        "keystore" => keystore,
        "keystore_password" => "keystore",
        "truststore" => truststore,
        "truststore_password" => "truststore",
        "ssl_certificate_verification" => true
      }
    end
  end
end
```

In this case it makes evident what changes in the fixture setup between the two cases.